server/config/swagger.js

```diff
@@ -0,0 +1,23 @@
+import swaggerJSDoc from 'swagger-jsdoc';
+
+// Swagger Definitions
+const swaggerDefinition = {
+  info: {
+    title: 'Author\'s Haven',
+    version: '1.0.0',
+    description: 'A Social platform for the creative at heart',
+  },
+  host: 'https://kingsmen-ah-backend-staging.herokuapp.com/',
+  basePath: 'api/api-docs'
```

Comment:
change to just `/api-docs` if possible

Reply:
@fidelisojeah done. Thank you for the feedback